crates/c-api/include/wasmtime/async.h

```diff
@@ -126,59 +135,58 @@ typedef struct wasmtime_async_continuation_t {
 /**
  * \brief Callback signature for #wasmtime_linker_define_async_func.
  *
- * This is a host function that returns a continuation to be called later. 
+ * This is a host function that returns a continuation to be called later.
  *
  * All the arguments to this function will be kept alive until the continuation
  * returns that it has errored or has completed.
  *
- * \param env user-provided argument passed to #wasmtime_linker_define_async_func
+ * \param env user-provided argument passed to
+ *        #wasmtime_linker_define_async_func
  * \param caller a temporary object that can only be used during this function
- * call. Used to acquire #wasmtime_context_t or caller's state
+ *        call. Used to acquire #wasmtime_context_t or caller's state
  * \param args the arguments provided to this function invocation
- * \param nargs how many arguments are provided
- * \param results where to write the results of this function
+ * \param nargs how many arguments are provided \param results where to write
```

Comment:
Might need some reformatting here

Reply:
Nice catch! Fixed. Thanks for working with me on this.